extensions/amp-instagram/1.0/component.type.js

```diff
@@ -24,9 +24,6 @@ var InstagramDef = {};
  *   shortcode: string,
  *   captioned: (boolean|undefined),
  *   title: (string|undefined),
- *   requestResize: (function(number):*|undefined),
```

Comment:
These still seem to be important component props. They are just implemented by the subcomponent. But in terms of API there are still there and must be passed.